global.json

```diff
@@ -3,7 +3,7 @@
     "Microsoft.Build.Traversal": "1.0.45"
   },
   "sdk": {
-    "version": "3.1.301",
+    "version": "5.0.100",
```

Comment:
Given that this just released are we sure we want to move the baseline to this version so quickly? I can understand starting to test this in CI but forcing everyone in the repo to move will cause some issues. 

@AlexGhiondea what are your thoughts here?

Reply:
Considering we are adding net5.0 targets to all test projects 5.0 SDK is required to be able to test locally.

Reply:
Lets ensure we have agreement of moving the baseline up but if we have any blockers we should consider allowing rollforward on major with some detection that will add net5.0 as a test target if we are running on that framework.

Reply:
Definitely. I will hold this PR off as long as needed to get everyone on board.